index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title aria-label="title">Graphic Designer's Summit Website</title>
+  </head>
+  <body>
+    <!-- Navigation Section  -->
+
+    <section class="home-container">
+    <header>
+      <nav class="top-header">
+        <ul class="top-nav">
+          <li class="top-item">
+            <a href="#" class="nav-link"
+              ><img src="img/facebook-icon.png" alt=""
+            /></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link"><img src="img/t-icon.png" alt=""/></a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">English</a>
+          </li>
+          <li class="top-item">
+            <a href="#" class="nav-link">login</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="navbar">
+        <a href="index.html" class="nav-branding"
+          ><img src="img/logo1.png" alt="logo image"
+        /></a>
+        <ul class="nav-menu nav-menu1">
+          <li class="nav-item home-href">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">Join</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">News</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-me" class="nav-link">
+              <span> GD Campaign</span></a
+            >
+          </li>
+        </ul>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <section class="first-container" aria-label="who I am">
+      <section class="introduction" aria-label="who I am">
+        <h3 class="">"Hello! Sharing world"</h3>
+        <div class="mask">
+          <h1>GRAPHIC DESIGNERS GLOBAL SUMMIT 2023</h1>
+        </div>
```

Comment:
- [ ] [optional] @Donmark2k great work implementing the changes requested, although the previous reviewer didn't point it out it would be nice if the layout matches the design given as shown below in the desktop view, 
![image](https://user-images.githubusercontent.com/61408860/210957235-eb51cd3c-d554-4146-8f17-fa15de1416a0.png)

Reply:
I have implemented this